package.json

```
@@ -26,5 +27,13 @@
   "lint-staged": {
     "*.{ts,js,md}": "prettier --write"
   },
-  "packageManager": "yarn@4.4.0"
+  "packageManager": "yarn@4.4.0",
+  "dependencies": {
+    "@types/react": "^19.0.0",
+    "@types/react-dom": "^19.0.0",
+    "bootstrap": "^5.3.3",
+    "react": "19.0.0",
+    "react-bootstrap": "^2.10.7",
+    "react-dom": "19.0.0"
+  }
```

These dependencies are not required in the base package.json of monorepo.
```suggestion
  "packageManager": "yarn@4.4.0"
```

They're added in `packages/frontend/package.json`